README.md

```diff
@@ -0,0 +1,55 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+ Microverse project
+
+> One paragraph statement about the project.
+
+![screenshot](./app_screenshot.png)
+
+Additional description about the project and its features.
+
+
+## Live Demo
+
+[Live Demo Link](https://livedemo.com)
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**This project was done today.It was done according to github flow.It's basically about practicing what i learnt in the previous lessons.There's not much coding.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+
+
+
+
+## Authors
+
+👤 **Author1**
```

Comment:
- Update your ``README`` file and change de default information and display your project information.